src/filter.scss

```diff
@@ -1,59 +1,75 @@
 .filter__title {
-    margin: 0 auto;
+    margin: 20px auto;
     text-align: center;
 
 }
 
 .filter {
-    margin: 20px auto;
+    margin: 10px auto;
     border: 1px solid gray;
-    width: 80%;
-    max-width: 1000px;
+    width: 91vw;
+    // max-width: 1000px;
     max-height: fit-content;
     display: flex;
     flex-direction: column;
+    animation: openUp 0.2s;
```

Comment:
Nicely done! Good initiative that we have the same type of animation throughout the webbsite.